terraform/cyhy_vulnscanner_acl_rules.tf

```diff
@@ -0,0 +1,65 @@
+# Allow ingress from public subnet (NAT gateway) via the Nessus UI and ssh ports
```

Comment:
Similarly to the comment at the beginning of `cyhy_portscanner_acl_rules.tf`, his comment should read, "Allow ingress from the public subnet (bastion) via...".

This is the one place where the reference to the Nessus UI port should remain, since we will want to be able to access that port via ssh port forwarding.

Reply:
👍